packages/manager/apps/pci-load-balancer/src/pages/create/steps/size/input/Label.component.tsx

```diff
@@ -2,31 +2,36 @@ import { useTranslation } from 'react-i18next';
 import { OsdsText } from '@ovhcloud/ods-components/react';
 import { ODS_TEXT_LEVEL, ODS_TEXT_SIZE } from '@ovhcloud/ods-components';
 import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
-import { useCatalogPrice } from '@ovh-ux/manager-react-components';
+import {
+  useCatalogPrice,
+  convertHourlyPriceToMonthly,
+} from '@ovh-ux/manager-react-components';
 import { TAddon } from '@/pages/create/store';
 
 export const LabelComponent = ({
   item,
-  isItemSelected,
 }: Readonly<{
   item: TAddon;
-  isItemSelected: boolean;
 }>) => {
   const { t: tCreate } = useTranslation('load-balancer/create');
 
-  const { getFormattedHourlyCatalogPrice } = useCatalogPrice();
+  const {
+    getFormattedHourlyCatalogPrice,
+    getFormattedMonthlyCatalogPrice,
+  } = useCatalogPrice(4);
```

Comment:
```suggestion
  } = useCatalogPrice(4);
```

What is "4" ? We can add constant for readability ?

Reply:
Is this more clear maybe ?
```suggestion
 const PRICE_DECIMAL_DIGIT = 4;
 
 useCatalogPrice(PRICE_DECIMAL_DIGIT);
```

Reply:
4 represents the number of digits after the decimal. I'll put it in a const yes